cmd/exchange_template/exchange_template.go

```diff
@@ -173,31 +172,38 @@ func makeExchange(exchangeDirectory string, configTestFile *config.Config, exch
 		{
 			Name:         "readme",
 			Filename:     "README.md",
-			TemplateFile: "readme_file.tmpl",
+			TemplateFile: "readme.tmpl",
```

Comment:
![image](https://github.com/user-attachments/assets/5dc8bf78-cba8-454f-9e23-94970752eeea)

Suggestion: We can purge contributors list as we can do this in documentation upgrade and assign based on git blame. But its in our commit history and front page anyway.